detectors/host/host_id_windows.go

```diff
@@ -0,0 +1,24 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build windows
+// +build windows
+
+package host // import "go.opentelemetry.io/contrib/detectors/host"
+
+import (
+	"golang.org/x/sys/windows/registry"
+)
+
+func getHostId() (string, error) {
+	key, err := registry.OpenKey(registry.LOCAL_MACHINE, `SOFTWARE\Microsoft\Cryptography`, registry.QUERY_VALUE)
```

Comment:
```suggestion
	key, err := registry.OpenKey(registry.LOCAL_MACHINE, `SOFTWARE\Microsoft\Cryptography`, registry.QUERY_VALUE|registry.WOW64_64KEY)
```

https://github.com/open-telemetry/opentelemetry-go/blob/main/sdk/resource/host_id_windows.go